Lib/test/test_os.py

```diff
@@ -3102,6 +3102,18 @@ def test_dup2(self):
         self.assertEqual(os.dup2(fd, fd3, inheritable=False), fd3)
         self.assertFalse(os.get_inheritable(fd3))
 
+        # dup2(fd, fd) must have no effect for a valid fd
+        # Issue #26935: Avoid failure due to a bionic bug
```

Comment:
```suggestion
        # bpo-26935: Avoid failure due to a bionic bug
```
I believe it's preferred to use `bpo-n` in code comments.